packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -372,6 +372,7 @@ export type CallCompositeIcons = {
     ParticipantItemScreenShareStart?: JSX.Element;
     VideoTileMicOff?: JSX.Element;
     LocalCameraSwitch?: JSX.Element;
+    AddPerson?: JSX.Element;
```

Comment:
This name needs more context given that we have a flat list of icons.

How about `ParticipantPaneAddPerson`?